src/pages/home.js

```diff
@@ -4,7 +4,15 @@ import { input } from '../components/input'
 export default (s,a) =>
   h('page', {}, [
     h('header', {}, input(s,a)),
-    h('search-', {}, s.search.map((item) =>
+    h('search-', {
+      oncreate: e => {
+        window.onscroll = () => {
```

Comment:
I'm wondering if here.. we should do `window.addEventListener('scroll', fn)` so then later we can then do `window.removeEventListener('scroll', fn)` when the `onremove` event fires. This also allows for other elements to register event listeners. At the moment this overwrite any other events that depend on `window.scroll`